---
lang: en
title: Software Development and Bioinformatics
title-prefix: Kim Rutherford
firstname: Kim
lastname: Rutherford
homepage: https://kmr.nz
email: kmr@kmr.nz
mobile: '+64 212508293'
address:
  city: Dunedin
  country: New Zealand
settings:
  protect-email: false
  protect-mobile: true
  protect-phone: true
  protect-fax: true
  protect-homepage: false
footer: |
  - [kmr.nz](https://kmr.nz)
  - [kimrutherford](https://github.com/kimrutherford)
  - [@kmr@mastodon.nz](https://mastodon.nz/@kmr)
  - [ORCID](https://orcid.org/0000-0001-6277-726X)
  - Skype: kim_rutherford
---

<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-55LZDGSCTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-55LZDGSCTQ');
</script>


Profile
-------

I am a software developer and bioinformatician.  For the last 25 years I have
been helping biologists to capture, analyse and interpret genomic data.

Skills and Experience
---------------------

* Software development
* Database administration and maintenance
* Genome assembly, annotation and analysis
* Processing, management and analysis of next generation sequencing data
* Systems administration including software and database installation and
  configuration

Work experience
---------------


<span class="nobr">June 2010</span> – <span class="nobr">Present</span>
:    **Programmer** *S. pombe genome database (PomBase)* \
     Department of Biochemistry, University of Cambridge

    Tasks
      ~ full stack development
      ~ bioinformatics support
      ~ system administration

    Major projects
      ~ [Canto](https://curation.pombase.org) - a community curation tool
      ~ [PomBase v2](https://www.pombase.org)
      ~ [JaponicusDB](https://www.japonicusdb.org)

    Secondments
      ~ FlyBase - 3 months customising
        [Canto](https://curation.pombase.org) for drosophila phenotypes
      ~ PHI-base - 6 months adapting Canto for pathogen-host
        interaction phenotype curation

<span class="nobr">Sept 2012</span> – <span class="nobr">Dec 2016</span>
:    **Scientific Officer** (part time) *Gemmell Group* \
     Department of Anatomy, University of Otago

    Tasks
      ~ bioinformatics support
      ~ genome and transcriptome assembly and analysis
      ~ bioinformatics mentoring and training
      ~ [tuatara](https://en.wikipedia.org/wiki/Tuatara) [genome project](https://www.nature.com/articles/s41586-020-2561-9)

<span class="nobr">Feb 2009</span> – <span class="nobr">June 2010</span>
:    **Computer Associate** *Baulcombe Group* \
     Department of Plant Sciences, University of Cambridge

    Tasks
      ~ bioinformatics support
      ~ creation of a short read analysis pipeline

<span class="nobr">Sept 2007</span> – <span class="nobr">Feb 2009</span>
:    **Programmer** *modENCODE Data Coordination Center* \
     Cambridge Systems Biology Centre

    Tasks
      ~ helped create the data warehouse for the modENCODE project: \
        [modMine](http://intermine.modencode.org/)

<span class="nobr">Feb 2004</span> – <span class="nobr">Sept 2007</span>
:    **Programmer** *FlyMine/InterMine group* \
     Department of Genetics, University of Cambridge

    Tasks
      ~ Java programming for the [InterMine](http://intermine.org) project

<span class="nobr">Oct 2001</span> – <span class="nobr">Feb 2004</span>
:    **Principal Computer Programmer** *Pathogen Sequencing Unit* \
     The Sanger Institute

    Tasks
      ~ programming and bioinformatics support

    Major projects
      ~ [Artemis](http://www.sanger.ac.uk/science/tools/artemis) - a genome analysis and annotation tool
      ~ [ACT](http://www.sanger.ac.uk/science/tools/artemis-comparison-tool-act) - a genome comparison viewer

<span class="nobr">Apr 1999</span> – <span class="nobr">Oct 2001</span>
:    **Senior Computer Programmer** *Pathogen Sequencing Unit* \
     The Sanger Institute

<span class="nobr">Sept 1998</span> – <span class="nobr">Apr 1999</span>
:    **Computer Programmer** *Pathogen Sequencing Unit* \
     The Sanger Institute


Publications
------------

<div style="float: right; font-size: 90%; padding: 0.2em 0 0 0;">
  <span style="display: inline-block"><span style="color: grey;">&#9679;</span> [Google Scholar](https://scholar.google.co.nz/citations?hl=en&user=63wyizYAAAAJ&view_op=list_works&sortby=pubdate)</span>
  <span style="display: inline-block"><span style="color: grey;">&#9679;</span> [ORCID](https://orcid.org/0000-0001-6277-726X)</span>
  <span style="display: inline-block"><span style="color: grey;">&#9679;</span> [Europe PMC](https://europepmc.org/authors/0000-0001-6277-726X)</span>
  <span style="display: inline-block"><span style="color: grey;">&#9679;</span> [ResearcherID](https://www.researcherid.com/rid/Y-7995-2018)</span>
  <span style="display: inline-block"><span style="color: grey;">&#9679;</span> [Scopus profile](https://www.scopus.com/authid/detail.uri?authorId=7007171631)</span>
</div>

Jan 2024
:   **PomBase, A Global Core Biodata Resource: Growth, Collaboration and Sustainability** \
    K.M. Rutherford, M. Lera-Ramírez, V. Wood \
    *GENETICS, in press*

Oct 2023
:   **Revised fission yeast gene and allele nomenclature guidelines for machine readability** \
    M. Lera-Ramírez, J. Bähler, J. Mata, K. Rutherford, C.S. Hoffman,
    S. Lambert, S. Oliferenko, S.G. Martin, K.L. Gould, L-L. Du,
    S.A. Sabatinos, S.L. Forsburg, O. Nielsen, P. Nurse, V. Wood \
    *GENETICS, iyad143 [DOI: 10.1093/genetics/iyad143](https://doi.org/10.1093/genetics/iyad143)*

July 2023
:   **A framework for community curation of interspecies interactions literature** \
    A. Cuzick, J. Seager, V. Wood, M. Urban, K. Rutherford, K.E. Hammond-Kosack \
    *eLife. [DOI: 10.7554/eLife.84658](https://doi.org/10.7554/eLife.84658)*

March 2023
:   **The Gene Ontology knowledgebase in 2023** \
    The Gene Ontology Consortium \
    *GENETICS. [DOI: 10.1093/genetics/iyad031](https://doi.org/10.1093/genetics/iyad031)*

Dec 2021
:   **JaponicusDB: Rapid deployment of a model organism database for
    an emerging model species** \
    K. Rutherford, M. Harris, S. Oliferenko, V. Wood \
    *GENETICS. [DOI: 10.1093/genetics/iyab223](https://doi.org/10.1093/genetics/iyab223)*

Dec 2021
:   **Fission stories: Using PomBase to understand _Schizosaccharomyces pombe_ biology** \
    M. Harris, K. Rutherford, J. Hayles, A. Lock, J. Bähler, S. Oliver, J. Mata, V. Wood \
    *GENETICS. [DOI: 10.1093/genetics/iyab222](https://doi.org/10.1093/genetics/iyab222)*

Nov 2021
:   **PHI-base in 2022: a multi-species phenotype database for Pathogen-Host Interactions** \
    M. Urban *et al.* \
    *NAR [DOI: 10.1093/nar/gkab1037](https://doi.org/10.1093/nar/gkab1037)*

Dec 2020
:   **The Gene Ontology resource: enriching a GOld mine** \
    The Gene Ontology Consortium \
    *Nucleic Acids Research. [DOI: 10.1093/nar/gkaa1113](https://doi.org/10.1093/nar/gkaa1113)*

Dec 2020
:   **A genome-wide investigation of adaptive signatures in
    protein-coding genes related to tool behaviour in New Caledonian
    and Hawaiian crows** \
    N. Dussex *et al.* \
    *Molecular Ecology. [DOI: 10.1111/mec.15775](https://doi.org/10.1111/mec.15775)*

Nov 2020
:   **Dense sampling of bird diversity increases power of comparative
    genomics** \
    S. Feng *et al.* \
    *Nature. [DOI: 10.1038/s41586-020-2873-9](https://doi.org/10.1038/s41586-020-2873-9)*

Sep 2020
:   **Term Matrix: A novel Gene Ontology annotation quality control
    system based on ontology term co-annotation patterns** \
    V. Wood *at al.* \
    *Open Biology. [DOI: 10.1098/rsob.200149](https://doi.org/10.1098/rsob.200149)*

Aug 2020
:   **The tuatara genome reveals ancient features of amniote evolution** \
    N. Gemmell, K. Rutherford, *et al.* \
    *Nature. [DOI: 10.1038/s41586-020-2561-9](https://doi.org/10.1038/s41586-020-2561-9)*

July 2020
:   **DNA from mollusc shell: a valuable and underutilised substrate for genetic analyses** \
    S. Ferreira, R. Ashby, G-J. Jeunen, K. Rutherford, C. Collins, E.V. Todd, N.J. Gemmell \
    *PeerJ. [DOI: 10.7717/peerj.9420](https://doi.org/10.7717/peerj.9420)*

Apr 2020
:   **Community curation in PomBase: enabling fission yeast experts
    to provide detailed, standardized, sharable annotation from
    research publications** \
    A. Lock, M. Harris, K. Rutherford, J. Hayles, V. Wood \
    *Database. [DOI: 10.1093/database/baaa028](https://doi.org/10.1093/database/baaa028)*

Nov 2019
:   **PHI-base: the pathogen–host interactions database** \
    M. Urban, A. Cuzick, J. Seager, V. Wood, K. Rutherford, S. Yagwakote Venkatesh,
    N. De Silva, M. Carbajo Martinez, H. Pedro, A.D. Yates, K. Hassani-Pak,
    K.E. Hammond-Kosack \
    *Nucleic Acids Research. [DOI: 10.1093/nar/gkz904](https://doi.org/10.1093/nar/gkz904)*

July 2019
:   **Stress, novel sex genes and epigenetic reprogramming orchestrate
    socially-controlled sex change** \
    E.V. Todd, O. Ortega-Recalde, H. Liu, M.S. Lamm, K.M. Rutherford,
    H. Cross, M.A. Black, O. Kardailsky, J.A. Graves, T.A. Hore,
    J.R. Godwin, N.J. Gemmell \
    *Science Advances. [DOI: 10.1126/sciadv.aaw7006](https://doi.org/10.1126/sciadv.aaw7006)*

July 2019
:   **Molecular structure of sauropsid β-keratins from tuatara (_Sphenodon punctatus_)** \
    D. Parry, R. Fraser, L. Alibardi, K. Rutherford, N. Gemmell \
    *Journal of Structural Biology. [DOI: 10.1016/j.jsb.2019.04.008](https://doi.org/10.1016/j.jsb.2019.04.008)*

Feb 2019
:   **Hidden in plain sight: What remains to be discovered in the
    eukaryotic proteome?** \
    V. Wood, A. Lock, M. Harris, K. Rutherford, J. Bahler, S Oliver \
    *Open Biology. [DOI: 10.1098/rsob.180241](https://royalsocietypublishing.org/doi/full/10.1098/rsob.180241)*

Nov 2018
:   **RNAcentral: a hub of information for non-coding RNA sequences** \
    The RNAcentral Constortium \
    *Nucleic Acids Research.  [DOI: 10.1093/nar/gky1034](https://doi.org/10.1093/nar/gky1034)*

Oct 2018
:   **PomBase 2018: user-driven reimplementation of the fission yeast
    database provides rapid and intuitive access to diverse,
    interconnected information** \
    A. Lock, K. Rutherford, M.A. Harris, J Hayles, S.G. Oliver, J. Bähler,
    V. Wood \
    *Nucleic Acids Research. [DOI: 10.1093/nar/gky961](https://doi.org/10.1093/nar/gky961)*

Oct 2018
:   **The Gene Ontology Resource: 20 years and still GOing strong** \
    The Gene Ontology Consortium \
    *Nucleic Acids Research.  [DOI: 10.1093/nar/gky1055](https://doi.org/10.1093/nar/gky1055)*

Aug 2018
:   **Evolutionary history of the podoplanin gene** \
    J. Renart, D. San Mauro, A. Agorreta, K. Rutherford, N. J. Gemmell,
    M Quintanill \
    *Gene Reports. [DOI: 10.1016/j.genrep.2018.08.005](https://doi.org/10.1016/j.genrep.2018.08.005)*

Aug 2018
:   **Reduced representation sequencing detects only subtle regional
    structure in a heavily exploited and rapidly recolonizing marine
    mammal species** \
    N. Dussex, H.R. Taylor, W.R. Stovall, K. Rutherford, K.G. Dodds,
    S.M. Clarke, N.J. Gemmell \
    *Ecology and Evolution. [DOI: 10.1002/ece3.4411](https://doi.org/10.1002/ece3.4411)*

Apr 2018
:   **Adipose Transcriptome Analysis Provides Novel Insights into Molecular
    Regulation of Prolonged Fasting in Northern Elephant Seal Pups** \
    B. Martinez, J. Khudyakov, K. Rutherford, D. Crocker, N. Gemmell, R. Ortiz \
    *Physiological Genomics. [DOI: 10.1152/physiolgenomics.00002.2018](https://doi.org/10.1152/physiolgenomics.00002.2018)*

Apr 2018
:   **_De novo_ draft assembly of the *Botrylloides leachii* genome
    provides further insight into tunicate evolution** \
    S. Blanchoud, K. Rutherford, L. Zondag, N. Gemmell, M Wilson \
    *Scientific Reports. [DOI: 10.1038/s41598-018-23749-w.](https://doi.org/10.1038/s41598-018-23749-w)*

Feb 2018
:   **Genetic sex assignment in wild populations using GBS data: a
    statistical threshold approach** \
    W. Stovall, H.R. Taylor, M. Black, S. Grosser, K. Rutherford, N.J. Gemmell \
    *Molecular Ecology Resources. [DOI: 10.1111/1755-0998.12767](https://doi.org/10.1111/1755-0998.12767)*

Dec 2017
:   **Analysis of the genome of the New Zealand giant collembolan
    (*Holacanthella duospinosa*) sheds light on hexapod evolution** \
    C. Wu, M.D. Jordan, R.D. Newcomb, N.J. Gemmell, S. Bank,
    K. Meusemann, P.K. Dearden, E.J. Duncan, S. Grosser,
    K. Rutherford, P.P. Gardner, R.N. Crowhurst, B. Steinwender,
    L.K. Tooman, M.I. Stevens, T.R. Buckley \
    *BMC Genomics. [DOI: 10.1186/s12864-017-4197-1.](https://doi.org/10.1186/s12864-017-4197-1)*

Dec 2017
:   **Identification of sex differences in zebrafish (*Danio rerio*)
    brains during early sexual differentiation and masculinisation
    using 17α-methyltestoterone** \
    S.L.J. Lee, J.A. Horsfield, M.A. Black, K.M. Rutherford, N.J. Gemmell \
    *Biology of Reproduction. [DOI: 10.1093/biolre/iox175](https://doi.org/10.1093/biolre/iox175)*

Nov 2017
:   **Female mimicry by sneaker males has a transcriptomic signature
    in both the brain and gonad in a sex changing fish** \
    E.V. Todd, H. Liu, M.S. Lamm, J.T. Thomas, K. Rutherford, K.C. Thompson,
    J.R. Godwin, N.J. Gemmell \
    *Molecular Biology and Evolution. [DOI: 10.1093/molbev/msx293](https://doi.org/10.1093/molbev/msx293)*

July 2017
:   **Male-female relatedness at specific SNP-linkage groups
    influences cryptic female choice in Chinook salmon (*Oncorhynchus
    tshawytscha*)** \
    C. Gessner, S.L. Johnson, P. Fisher, S. Clarke, K. Rutherford,
    J. Symonds, N.J. Gemmell \
    *Proc. R. Soc. B. [DOI: 10.1098/rspb.2017.0853.](https://doi.org/10.1098/rspb.2017.0853)*

July 2017
:   **Histological and transcriptomic effects of
    17α-methyltestosterone on zebrafish gonad development** \
    S.L.J. Lee, J.A. Horsfield, M.A. Black, K. Rutherford, A. Fisher,
    N.J. Gemmell \
    *BMC Genomics. [DOI: 10.1186/s12864-017-3915-z](https://doi.org/10.1186/s12864-017-3915-z)*

June 2017
:   **PomBase – the scientific resource for fission yeast** \
    V. Wood, A. Lock, K. Rutherford, M.A. Harris \
    *Methods in Molecular Biology. [DOI: 10.17863/CAM.12124](https://doi.org/10.17863/CAM.12124)*

Jan 2017
:   **PHI-base: A new interface and further additions for the multi-species
    pathogen-host interactions database.** \
    Urban, M., Alayne Cuzick, A., Rutherford, K., Irvine, A., Pedro, H., Pant,
    R., Sadanadan, V., Khamari, L., Billal, S., Mohanty S.,
    and. Hammond-Kosack, K.E. \
    *Nucleic Acids Research. [DOI: 10.1093/nar/gkw1089](https://doi.org/10.1093/nar/gkw1089)*

Nov 2016
:   **The Gene Ontology Consortium; Expansion of the Gene Ontology
    knowledgebase and resources.** \
    The Gene Ontology Consortium \
    *Nucleic Acids Research. [DOI: 10.1093/nar/gkw1108](https://doi.org/10.1093/nar/gkw1108)*

Feb 2016
:   **Uncovering the pathways underlying whole body regeneration in a chordate
    model, *Botrylloides leachi* using de novo transcriptome analysis** \
    L. Zondag, K. Rutherford, N. Gemmell and M. Wilson \
    *BMC Genomics. [DOI: 10.1186/s12864-016-2435-6](https://doi.org/10.1186/s12864-016-2435-6)*

Nov 2015
:    **Large-scale transcriptome sequencing reveals novel expression patterns
     for key sex-related genes in a sex-changing fish** \
     H. Liu, M. Lamm, K. Rutherford, M. Black, J. Godwin and N. Gemmell \
     *Biology of Sex Differences. [DOI: 10.1186/s13293-015-0044-8](https://dx.doi.org/10.1186%2Fs13293-015-0044-8)*

Jan 2015
:    **Gene Ontology Consortium: going forward** \
     The Gene Ontology Consortium \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gku1179](https://doi.org/10.1093/nar/gku1179)*

Dec 2014
:    **Molecular evolution of Dmrt1 accompanies change of sex-determining
     mechanisms in reptilia** \
     D. E. Janes *et al.* \
     *Biology Letters. [DOI: 10.1098/rsbl.2014.0809](https://doi.org/10.1098/rsbl.2014.0809)*

Oct 2014
:    **PomBase 2015: updates to the fission yeast database** \
     M. McDowall, M. Harris, A. Lock, K. Rutherford, D. Staines, J. Bähler,
        P. Kersey, S. Oliver and V. Wood \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gku1040](https://doi.org/10.1093/nar/gku1040)*

June 2014
:    **Improving functional annotation for industrial microbes: A case study with _Pichia pastoris_** \
     D. Dikicioglu, V. Wood, K. Rutherford, M. McDowall, S. Oliver \
     *Trends in Biotechnology. [DOI: 10.1016/j.tibtech.2014.05.003](https://doi.org/10.1016/j.tibtech.2014.05.003)*

Feb 2014
:    **Canto: An online tool for community literature curation** \
     K. Rutherford, M. A. Harris, A. Lock, S. G. Oliver and V. Wood \
     *Bioinformatics. [DOI: 10.1093/bioinformatics/btu103](https://doi.org/10.1093/bioinformatics/btu103)*

Jan 2013
:    **Gene ontology annotations and resources.** \
     Gene Ontology Consortium \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gks1050](https://doi.org/10.1093/nar/gks1050)*

Sep 2012
:    **InterMine: a flexible data warehouse system for the integration and
     analysis of heterogeneous biological data** \
     R.N. Smith, J. Aleksic, D. Butano, A. Carr, S. Contrino, F. Hu, M. Lyne, R. Lyne, A. Kalderimis, K. Rutherford, R. Stepan, J. Sullivan, M. Wakeling, X. Watkins, G. Micklem \
     *Bioinformatics. [DOI: 10.1093/bioinformatics/bts577](https://doi.org/10.1093/bioinformatics/bts577)*

Jan 2012
:    **modMine: flexible access to modENCODE data.** \
     S. Contrino, R.N. Smith, D. Butano, A. Carr, F. Hu, R. Lyne,
     K. Rutherford, A. Kalderimis, J. Sullivan,
     S. Carbon, E.T. Kephart, P. Lloyd, E.O. Stinson, N.L. Washington,
     M.D. Perry, P. Ruzanov, Z. Zha, S.E. Lewis, L.D. Stein, G. Micklem \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gkr921](https://doi.org/10.1093/nar/gkr921)*

Jan 2012
:    **PomBase: a comprehensive online resource for fission yeast.** \
     V. Wood, M.A. Harris, M.D. McDowall, K. Rutherford, B.W. Vaughan, D.M. Staines, M. Aslett, A. Lock, J. Bähler, P.J. Kersey, S.G. Oliver \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gkr853](https://doi.org/10.1093/nar/gkr853)*

Nov 2011
:    **The Gene Ontology: enhancements for 2011.** \
     Gene Ontology Consortium. \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gkr1028](https://doi.org/10.1093/nar/gkr1028)*

Dec 2010
:    **Identification of Functional Elements and Regulatory Circuits by *Drosophila*
     modENCODE.** \
     The modENCODE Consortium \
     *Science. [DOI: 10.1126/science.1198374](https://doi.org/10.1126/science.1198374)*

Dec 2010
:    **Integrative Analysis of the *C. elegans* Genome by the modENCODE Project.** \
     The modENCODE Consortium \
     *Science. [DOI: 10.1126/science.1196914](https://doi.org/10.1126/science.1196914)*

July 2007
:    **FlyMine: an integrated database for *Drosophila* and *Anopheles* genomics.** \
     R. Lyne *et al.* \
     *Genome Biology. [DOI: 10.1186/gb-2007-8-7-r129](https://doi.org/10.1186/gb-2007-8-7-r129)*

Sep 2005
:    **WebACT - An online companion for the Artemis Comparison Tool** \
     J. Abbott, D. Aanensen, K. Rutherford, S. Butcher and B. Spratt \
     *Bioinformatics. [DOI: 10.1093/bioinformatics/bti601](https://doi.org/10.1093/bioinformatics/bti601)*

Aug 2005
:    **ACT: the Artemis comparison tool.** \
     T. Carver, K. Rutherford, M. Berriman, M-A. Rajandream, B.. Barrell and J. Parkhill \
     *Bioinformatics. [DOI: 10.1093/bioinformatics/bti553](https://doi.org/10.1093/bioinformatics/bti553)*

June 2005
:    **A Human-Curated Annotation of the *Candida albicans* Genome** \
     B.R. Braun *et al.* \
     *PLoS Genetics. [DOI: 10.1371/journal.pgen.0010001](https://doi.org/10.1371/journal.pgen.0010001)*

Jan 2005
:    **A comprehensive survey of the *Plasmodium* life cycle by genomic,
     transcriptomic, and proteomic analyses** \
     N. Hall *et al.* \
     *Science. [DOI: 10.1126/science.1103717](https://doi.org/10.1126/science.1103717)*

Sep 2004
:    **Genomic plasticity of the causative agent of melioidosis, *Burkholderia
     pseudomallei*.** \
     M. Holden *et al.* \
     *Proc Natl Acad Sci. [DOI: 10.1073/pnas.0403302101](https://doi.org/10.1073/pnas.0403302101)*

June 2004
:    **Complete genomes of two clinical *Staphylococcus aureus* strains:
     Evidence for the rapid evolution of virulence and drug resistance.** \
     M. Holden *et al.* \
     *Proc Natl Acad Sci. [DOI: 10.1073/pnas.0402521101](https://doi.org/10.1073/pnas.0402521101)*

Jan 2004
:    **GeneDB: a resource for prokaryotic and eukaryotic organisms.** \
     Hertz-Fowler C *et al.* \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gkh007](https://doi.org/10.1093/nar/gkh007)*

Nov 2003
:    **The complete genome sequence and analysis of *Corynebacterium diphtheriae*
     NCTC13129. ** \
     A. Cerdeño‐Tárraga *et al.* \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gkg874](https://doi.org/10.1093/nar/gkg874)*

Aug 2003
:    **The DNA sequence of chromosome I of an African trypanosome: Gene
     content, chromosome organisation, recombination and polymorphism** \
     N. Hall *et al.* \
     *Nucleic Acids Research. [DOI: 10.1093/nar/gkg674](https://doi.org/10.1093/nar/gkg674)*

June 2003
:    **Viewing and annotating sequence data with Artemis.** \
     M. Berriman and K. Rutherford \
     *Briefing in Bioinformatics. [DOI: 10.1093/bib/4.2.124](https://doi.org/10.1093/bib/4.2.124)*

Oct 2002
:    **Genome sequence of the human malaria parasite *Plasmodium falciparum*.** \
     J. Malcolm *et al.* \
     *Nature. [DOI: 10.1038/nature01097](https://doi.org/10.1038/nature01097)*

Oct 2002
:    **Sequence of *Plasmodium falciparum* chromosomes 1, 3–9 and 13.** \
     N. Hall *et al.* \
     *Nature. [DOI: 10.1038/nature01095](https://doi.org/10.1038/nature01095)*

May 2002
:    **Complete genome sequence of the model actinomycete *Streptomyces coelicolor*
     A3(2).** \
     S. Bentley *et al.* \
     *Nature. [DOI: 10.1038/417141a](https://doi.org/10.1038/417141a)*

Feb 2002
:    **The genome sequence of *Schizosaccharomyces pombe*.** \
     V. Wood *et al.* \
     *Nature. [DOI: 10.1038/nature724](https://doi.org/10.1038/nature724)*

Oct 2001
:    **Complete genome sequence of a multiple drug resistant *Salmonella
     enterica* serovar Typhi CT18.** \
     J. Parkhill *et al.* \
     *Nature. [DOI: 10.1038/35101607](https://doi.org/10.1038/35101607)*

Oct 2001
:    **Genome sequence of *Yersinia pestis*, the causative agent of plague.** \
     J. Parkhill *et al.* \
     *Nature. [DOI: 10.1038/35097083](https://doi.org/10.1038/35097083)*

June 2001
:    **A Re-annotation of the *Saccharomyces cerevisiae* Genome.** \
     V. Wood, K. M. Rutherford, A. Ivens, M-A Rajandream and B. Barrell \
     *Comparative and Functional Genomics. [DOI: 10.1002/cfg.86](https://pubmed.ncbi.nlm.nih.gov/18628908/)*

Feb 2001
:    **Massive gene decay in the leprosy bacillus.** \
     S. Cole *et al.* \
     *Nature. [DOI: 10.1038/35059006](https://doi.org/10.1038/35059006)*

Oct 2000
:    **Artemis: sequence visualisation and annotation.** \
     K. Rutherford, J. Parkhill, J. Crook, T. Horsnell, P. Rice,
        M-A. Rajandream and B. Barrell \
     *Bioinformatics. [DOI: 10.1093/bioinformatics/16.10.944](https://doi.org/10.1093/bioinformatics/16.10.944)*

Sep 2000
:    **Analysis of 114 kb of DNA sequence from fission yeast
     chromosome 2 immediately centromere‐distal to _his5_.** \
     Z. Xiang *et al.* \
     *Yeast. [ PMID:11054821](https://doi.org/10.1002/1097-0061\(200011\)16:15%3C1405::AID-YEA625%3E3.0.CO;2-H)*

Mar 2000
:    **Complete DNA sequence of a serogroup A strain of *Neisseria meningitidis* Z2491.** \
     J. Parkhill *et al.* \
     *Nature. [DOI: 10.1038/35006655](https://doi.org/10.1038/35006655)*

Feb 2000
:    **The genome sequence of the food-borne pathogen *Campylobacter jejuni*
     reveals hypervariable sequences.** \
     J. Parkhill *et al.* \
     *Nature. [DOI: 10.1038/35001088](https://doi.org/10.1038/35001088)*


Book Chapters
-------------

May 2018
:   **PomBase: The Scientific Resource for Fission Yeast** \
    "Eukaryotic Genomic Databases : Methods and Protocols" \
    A Lock, K Rutherford,  M.A. Harris, V. Wood \
    *Methods in Molecular Biology. [DOI: 10.1007/978-1-4939-7737-6_4](https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4)*

<span style="display: none">
<a rel="me" href="https://mastodon.nz/@kmr">Mastodon</a>
</span>
